date.go

```diff
@@ -129,13 +137,20 @@ func (d *dateObject) isSet() bool {
 
 func (d *dateObject) unset() {
```

Comment:
Should the `isSet()` function above also check if `__wrapped` is defined? If not, `export()` will return `nil` even when `__wrapped` is set. We might have to change a few other places where we reference `d.mesc` in the `builtin_date.go` file (e.g. the `dateproto_getTime` function) where we directly access `d.msec` without checking if `__wrapped` is defined.